client/components/QuestionsAndAnswers/AnswerItem.jsx

```diff
@@ -1,50 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-const AnswerItem = ({ answer }) => (
-  <AnswerContainer data-testid='answerContainer'>
-    <AnswerBody>
-      A:
-      {' '}
-      {answer.body}
-      {' '}
-    </AnswerBody>
-    <AnswerDetails>
-      <AnswererName>
-        by
-        {' '}
-        {answer.answerer_name}
-        {' '}
-      </AnswererName>
-      <AnswerDate>
-        {new Date(answer.date).toLocaleDateString('en-US', {year: 'numeric', month: 'long', day: 'numeric'})}
-      </AnswerDate>
-      <AnswerHelpfulness>
-        Helpful?
-        {'   '}
-      </AnswerHelpfulness>
-      <UnderLineYes onClick={() => console.log('is this answer helpful?')}>
-        {'   '}
-        Yes
-        {' '}
-      </UnderLineYes>
-      <AnswerHelpfulness>
-        ({answer.helpfulness})
-      </AnswerHelpfulness>
-      <Divide>
-        |
-      </Divide>
-      <Report onClick={() => console.log('report this answer?')}>
-        Report
-      </Report>
-    </AnswerDetails>
-  </AnswerContainer>
-);
+const AnswerItem = ({ answer, index, updateAnswerHelpfulness }) => {
+  const [isHelpful, setIsHelpful] = useState(false);
+
+  const handleAnswerHelpfulness = () => {
+    console.log('clicked -------');
+    if (!isHelpful) {
+      setIsHelpful(true);
+      updateAnswerHelpfulness(index);
+    }
+  };
+
+  return (
+    <AnswerContainer data-testid="answerContainer">
+      <AnswerBody>
+        A:
+      </AnswerBody>
+      <AnswerBody>
```

Comment:
Consider wrapping <AnswerBody> once around A:{answer.body} to reduce duplicate code. Can add a <p> around {answer.body} if you want to break onto a separate line. Just something to consider, does not need to be changed before merging!

Reply:
thanks for pointing it out, will refactor to reduce the duplicate code